app/controllers/ideas_controller.rb

```diff
@@ -88,7 +103,8 @@ def idea_params
         :assigned_user_id,
         :participation_level,
         :status,
-        :review_date
+        :review_date,
```

Comment:
You can set the valid params depending on what action we are on. `params[:action]` will tell you. If we are making a new idea, we probably don't want to allow many of these, but if we're updating then we do.